servers/rendering/renderer_rd/shaders/samplers_inc.glsl

```diff
@@ -0,0 +1,12 @@
+layout(set = 0, binding = 1000) uniform sampler SAMPLER_NEAREST_CLAMP;
```

Comment:
@RandomShaper will this play nice with D3D12? I'm a bit worried about setting bindings as high as 1000 on APIs that don't have the same set/binding model

Reply:
Is there a way for me to test the Dx12 path?

Reply:
@ChibiDenDen You can checkout https://github.com/godotengine/godot/pull/70315, apply your changes on top and follow the build instructions in the PR description.

Reply:
Ran the dx12 branch and cherry-picked my change, it seems to work fine (I get an error "Heap walker is at EOF" constantly with or without my change)
![image](https://github.com/godotengine/godot/assets/11915378/db7c4784-ef2d-40b9-865a-f60a890a9820)

Reply:
Double checked with a texture mapped this time.
Dx12 is broken with this change.

Will update if I find a solution

Reply:
This is solved by either lowering GODOT_NIR_BINDING_MULTIPLIER

-static const uint32_t GODOT_NIR_BINDING_MULTIPLIER = 100000;
+static const uint32_t GODOT_NIR_BINDING_MULTIPLIER = 10000;

or lowering the binding numbers (lets say from 1000 to 100)

Screenshot from dx12 with texture:
![image](https://github.com/godotengine/godot/assets/11915378/bae08039-4bd6-45e0-bb0c-587526c536f2)

Reply:
`100000` was a very conservative one. `10000` should be fine, too.